test/e2e/load.go

```diff
@@ -113,8 +114,17 @@ var _ = framework.KubeDescribe("Load capacity", func() {
 		itArg := testArg
 
 		It(name, func() {
+			// Create a number of namespaces.
+			namespaceCount := (nodeCount + 249) / 250
```

Comment:
Is one namespace for every 250 Nodes a guessed value, or there's some reasoning behind it? Whatever this is please wrap the namespace creation in a function similar to one creating configs for RCs.